source/tutorials/learning-journey/packaging-existing-software.md

```diff
@@ -46,12 +46,21 @@ As you progress through this tutorial, you will update this several times, addin
 ### Hello, World!
 GNU Hello is an implementation of the "hello world" program, with source code accessible [from the GNU Project's FTP server](https://ftp.gnu.org/gnu/hello/).
 
-To begin, you should add a `name` attribute to the set passed to `mkDerivation`; every derivation needs a name, and Nix will throw `error: derivation name missing` without one.
+Every derivation needs a package name and version, so to begin, you should add a `pname` attribute and a `version` string to the set passed to `mkDerivation`.
```

Comment:
Nitpick: both `pname` and `version` are attributes. Referring to one as an attribute and the other as a string is slightly misleading.

Reply:
```suggestion
Every derivation needs a package name and version, so to begin, you should add `pname` and `version` attributes to the set passed to `mkDerivation`, setting these to appropriate strings.
```

Good point. How about this?

Reply:
Looks good to me 👍